app/validation/error_messages.py

```diff
@@ -10,4 +10,5 @@
     'INTEGER_TOO_LARGE': _('This number is too big.'),
     'INVALID_DATE_RANGE_TO_BEFORE_FROM': _("The 'period to' date cannot be before the 'period from' date."),
     'INVALID_DATE_RANGE_TO_FROM_SAME': _("The 'period to' date must be different to the 'period from' date."),
+    'MAX_LENGTH_EXCEEDED': _("Your answer is too long, it has to be less than %%(max)d characters"),
```

Comment:
I think we can avoid this double escaping `%%` by dropping the `_` gettext call for all of these strings as WTForms passes all error messages through gettext.